v2/azure-arm.yaml

```
@@ -213,6 +213,60 @@ typeTransformers:
     property: Type
     remove: true
     because: it is supposed to be marked ReadOnly in Swagger
+  - group: network
+    version: "*"
+    name: DnsZone_Spec
+    property: Etag
+    remove: true
+    because: it is supposed to be marked ReadOnly in Swagger
```

The values here are multi-valued, so you can combine these together:

```suggestion
  - group: network
    version: "*"
    name: DnsZone_Spec; DnsZones_AAAA_Spec; DnsZones_CAA_Spec; etc ...
    property: Etag
    remove: true
    because: it is supposed to be marked ReadOnly in Swagger
```

See the interface [`StringMatcher`](https://github.com/Azure/azure-service-operator/blob/main/v2/tools/generator/internal/config/string_matcher.go) and it's implementations for details.

---

oohh.. This is something I did not realise. Thanks, I'll update.